endpoints/openrtb2/auction_test.go

```diff
@@ -1047,40 +1139,6 @@ func TestStoredRequests(t *testing.T) {
 	}
 }
 
-func TestStoredRequestsVideoErrors(t *testing.T) {
-	deps := &endpointDeps{
```

Comment:
Did we move the error tests elsewhere? Or do these scenarios no longer apply?

Reply:
This test is not relevant anymore, now only unmarshal error can be returned. I added test for your another comment and it covers this case.